exporter/collector/config.go

```diff
@@ -59,6 +59,16 @@ type MetricConfig struct {
 	// Buffer size for the channel which asynchronously calls CreateMetricDescriptor. Default
 	// is 10.
 	CreateMetricDescriptorBufferSize int `mapstructure:"create_metric_descriptor_buffer_size"`
+
+	// If true, the exporter will copy OTel's service.name, service.namespace, and
+	// service.instance.id resource attributes into the GCM timeseries metric labels. This
+	// option is recommended to avoid writing duplicate timeseries against the same monitored
+	// resource. Default is true.
+	IncludeServiceResourceAttributes bool `mapstructure:"include_service_resource_attributes"`
```

Comment:
I don't think we need two config options to gate this behavior... `include_service_resource_attributes=true` seems to be equivalent to `resource_filter=service.*`.  Also, this should probably be a list?  What if I want my service.* and my host.*, but not others?

Reply:
Relevant part of the spec on these:

> [[3]: MUST be unique for each instance of the same service.namespace,service.name pair (in other words service.namespace,service.name,service.instance.id triplet MUST be globally unique). The ID helps to distinguish instances of the same service that exist at the same time (e.g. instances of a horizontally scaled service).](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.8.0/specification/resource/semantic_conventions#:~:text=%5B3%5D%3A%20MUST%20be%20unique%20for%20each%20instance%20of%20the%20same%20service.namespace%2Cservice.name%20pair%20(in%20other%20words%20service.namespace%2Cservice.name%2Cservice.instance.id%20triplet%20MUST%20be%20globally%20unique).)


So other service attributes (just `service.version` atm) are not needed for uniqueness. You could do `service.name|service.instance.id|service.namespace` but it would be easy to accidentally override this behavior when you specify a `resource_filter`. That was my thinking for separating it.

Reply:
@dashpole I think we are on the same page with this now. If so, can you resolve?